### DISCUSSION TITLE
How do I only generate args/returns table?

### DISCUSSION THREAD

**Discussion by Archmonger** [mkdocstrings/mkdocstrings#465] [Q&A] [answered]
There are certain circumstances where I only want to embed the args/returns, and no title/description/etc.

Is there any way of doing this?

**Comment by pawamoy** [comment]
Not currently I'm afraid. I'm open to suggestions/specifications to allow such functionality!

**Reply by Archmonger** [reply]
To fit the current design, it's easiest to just add a bunch of toggles.

- `show_title`
- `show_description`
- `show_args`
- `show_kwargs`
- `show_returns`
- `show_root_heading`
- `show_source`

**Reply by pawamoy** [reply]
Interesting. I'll think about it. It will be easier for me to not forget your request if you open a proper issue with your suggestion. Could you do that?

**Reply by pawamoy** [reply, marked as answer]
This was added in v0.9.0: https://mkdocstrings.github.io/python/changelog/#090-2023-04-03, https://mkdocstrings.github.io/python/usage/configuration/docstrings/
